kahuna/public/js/components/gr-panel-button/gr-panel-button.js

```diff
@@ -29,7 +30,24 @@ panelButton.directive('grPanelButton', [function() {
         scope: {
             panel: '=grPanel',
             position: '@grPosition',
-            name: '@grName'
+            name: '@grName',
+            icon: '@grIcon'
+        }
+    };
+}]);
+
+panelButton.directive('grPanelButtonSmall', [function() {
```

Comment:
Could it be done with css then? (though not sure if that'll be clearer or not...)